language-support/hs/bindings/build_packages.sh

```diff
@@ -1,165 +0,0 @@
-#!/usr/bin/env bash
```

Comment:
I would much rather see this be updated than move to a more manual process.

Reply:
Yup!
Although the script is call `export-package.sh` now